BookReader/BookReader.css

```diff
@@ -1030,10 +1030,16 @@ i.BRicon {
 .BRfooter button:hover .tooltip {
   display: block;
 }
+.BRfooter button.viewmode {
```

Comment:
this is a built file, we need to remove this bc it gets generated during versioning. let's get this out of the PR.